pkg/cloud/services/compute/instance.go

```diff
@@ -95,7 +95,7 @@ func (s *Service) normalizePortTarget(port *infrav1.PortOpts, openStackCluster *
 				if err != nil {
 					// Multiple matches might be ok later when we restrict matches to a single network
 					if errors.Is(err, networking.ErrMultipleMatches) {
-						s.scope.Logger().V(4).Info("Can't infer network from subnet %d: %s", i, err)
+						s.scope.Logger().V(4).Info("Couldn't infer network from subnet", "subnet", i, "err", err)
```

Comment:
Do we have any other instances where we're passing an error in a structured log? Do we have a standard name for them already?

Reply:
Also it doesn't seem like `i` is the subnet here. It's the index of the `Port`'s `FixedIPs`, we should rather use `fixedIP.Subnet` here.

Reply:
I think we should use "err" instead of "error" according to "With the exception of acronyms like "IP" and the standard "err", don't shorten names. " from this structured logging migration guideline(https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md).

Reply:
There is another instance like this in this file. On line 261: `s.scope.Logger().V(4).Error(err, "Failed to clean up ports after failure")`. I'm not sure if we already have a standard name for them.

Reply:
For the naming issue, I had a discussion with @mdbooth last week. We think `subnetIndex` is a proper choice. Sorry for not updating it in this PR. FixedIPs is a weird naming thing from OpenStack which really means subnets.